### PR TITLE
Amidst (new Cask)

### PR DESCRIPTION
**Pull request by JMoVS**
New cask to install Amidst app via homebrew cask

### CODE REVIEW

File: Casks/amidst.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'amidst' do
+  version '4.0'
+  sha256 '921300b83acf3a09f72b9adc1abeea6ce9975438b42814ae40c27e9debf13262'
+
+  url 'https://github.com/toolbox4minecraft/amidst/releases/download/v4.0/amidst-v4-0.zip'
```
Comment (victorpopkov) [inline Casks/amidst.rb]: You can use string interpolation with [version methods](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/version.md) here. An [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md) also should be specified with appropriate `checkpoint`:
```ruby
  url "https://github.com/toolbox4minecraft/amidst/releases/download/v#{version}/amidst-v#{version.dots_to_hyphens}.zip"
  appcast 'https://github.com/toolbox4minecraft/amidst/releases.atom',
           checkpoint: '84beb95511a46c011ddd2f72ab75c6cf56624d6baf59b85288ab19830620a864'
```

File: Casks/amidst.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name 'amidst'
+  name 'Amidst'
```
Comment (victorpopkov) [inline Casks/amidst.rb]: Only one name would be enough. [We don’t intend to become a discoverability service](https://github.com/caskroom/homebrew-cask/issues/4323#issuecomment-67257513). [It is not part of the goals of homebrew-cask](https://github.com/caskroom/homebrew-cask/issues/3817#issuecomment-39868330), so descriptions are currently not supported, especially in the `name` stanza:
```
  name 'Amidst'
```

Comment (JMoVS) [inline Casks/amidst.rb]: I didn't know whether finding it is case-sensitive or not ;-)

File: Casks/amidst.rb (hunk 1 above)
Comment (JMoVS) [inline Casks/amidst.rb]: hmm ok. I can change that, but will that mean that every time a new release is published, this cask will need an update?

Also, this app has its own update mechanism (that I haven't tested yet with a cask install), are users supposed to update via homebrew-cask or via in-app?

**Comment by victorpopkov** [comment]
Thank you for the contribution. It was merged directly as 9da1d90c7f2fbb4dd8e931d9a3255e13a8aee41c to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**